bin/public/js/term.js

```diff
@@ -1328,7 +1328,8 @@ Terminal.prototype.refresh = function(start, end) {
                 out += 'font-weight:bold;';
               }
               // See: XTerm*boldColors
-              if (fg < 8) fg += 8;
+              if(this.options.boldAsBright > 0 && fg < 8)
```

Comment:
Done - chjj/term.js#105
term.js master hasn't seen much activity though.

If we really want to stay in sync with the original projects better, we should refactor to use npm and a bundler like webpack